enclave-manager/server/server.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	listenPort                = 8081
+	listenPort                = 9081
```

Comment:
@leoporoli this will break a good bit of stuff. It needs to be 8081

Reply:
oh! yes sorry, I changed it for testing purposes but I forgot to revert it